### PR TITLE
Correct authn policy for health check demo

### PR DESCRIPTION
**Pull request by diemtvu**
* Namespace-wide authn policy must be named `default`.
* Also, add destinationrule that is needed to go with the example authn policy.

### CODE REVIEW

File: content/docs/tasks/traffic-management/app-health-check/index.md [hunk 1]
```diff
@@ -46,18 +46,34 @@ The number '0' in the 'RESTARTS' column means liveness probes worked fine. Readi
 
 ### Mutual TLS enabled
 
-Run this command to enable mutual TLS for services in the default namespace:
+Run these commands to enable mutual TLS for services in the default namespace. Note that you need both authentication policy and
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/app-health-check/index.md]: Consider using a numbered list for procedures. Having multiple commands and steps in a single paragraph makes translation harder, adds complexity to the content, and makes maintaining the content, updating it and correcting it, a lot harder. Consider these edits:
To enable mutual TLS for services in the default namespace, you must configure an authentication policy and a destination rule. Follow these steps to complete the configuration:

1. To configure the authentication policy, run:
    {{< text bash >}}
    $ cat <<EOF | istioctl create -f -
     apiVersion: "authentication.istio.io/v1alpha1"
     kind: "Policy"
     metadata:
       name: "default"
       namespace: "default"
     spec:
       peers:
       - mtls: {}
    EOF
    {{< /text >}}

1. To configure the destination rule, run:

    {{< text bash >}}
    $ cat <<EOF | istioctl create -f -
    apiVersion: "networking.istio.io/v1alpha3"
    kind: "DestinationRule"
    metadata:
      name: "default"
      namespace: "default"
    spec:
      host: "*.default.svc.cluster.local"
      trafficPolicy:
        tls:
          mode: ISTIO_MUTUAL
    EOF
    {{< /text >}}

Remember to indent the code blocks with four spaces for the list to work properly.

File: content/docs/tasks/traffic-management/app-health-check/index.md [hunk 2]
```diff
@@ -76,19 +92,11 @@ This section shows how to configure health checking with the HTTP request option
 
 ### Mutual TLS is disabled
 
-Run this command to remove the mutual TLS policy:
+Run this command to remove the mutual TLS policy and corresponding destination rule:
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/app-health-check/index.md]: See my previous comment.
These task would contain two steps, one for each command:

1. To remove remove the mutual TLS policy, run:

    {{< text bash >}}
    $ kubectl delete policies default
    {{< /text >}}

1. To remove remove the corresponding destination rule, run:

    {{< text bash >}}
    $ kubectl delete destinationrules default
    {{< /text >}}

File: content/docs/tasks/traffic-management/app-health-check/index.md [hunk 3]
```diff
@@ -107,17 +115,33 @@ liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
 
 ### Mutual TLS is enabled
 
-Run this command to enable mutual TLS for services in the default namespace:
+Run these commands to enable mutual TLS for services in the default namespace:
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/app-health-check/index.md]: See previous comments. Please split these content into two numbered steps.

File: content/docs/tasks/traffic-management/app-health-check/index.md (hunk 1 above)
Comment (diemtvu) [reply content/docs/tasks/traffic-management/app-health-check/index.md]: Done

File: content/docs/tasks/traffic-management/app-health-check/index.md (hunk 2 above)
Comment (diemtvu) [reply content/docs/tasks/traffic-management/app-health-check/index.md]: Done

File: content/docs/tasks/traffic-management/app-health-check/index.md (hunk 3 above)
Comment (diemtvu) [reply content/docs/tasks/traffic-management/app-health-check/index.md]: Done

**Pull request merged**